src/main/java/nl/knaw/dans/ingest/core/service/mapper/DepositToDvDatasetMetadataMapper.java

```diff
@@ -261,6 +264,18 @@ void processMetadataBlock(boolean deduplicate, Map<String, MetadataBlock> fields
         fields.put(title, block);
     }
 
+    private MetadataField compoundBuild(CompoundFieldBuilder compoundFieldBuilder) {
+        // TODO rewrite in the dataverse library
+        CompoundField compoundField = compoundFieldBuilder.build();
+        if (compoundField.isMultiple())
+            return compoundField;
+        else return new MetadataField(compoundField.getTypeClass(),compoundField.getTypeName(),false) {
+            public Map<String, SingleValueField> getValue() {
+                return compoundField.getValue().get(0);
```

Comment:
Perhaps a check to see if getValue() returns null would be useful, or is it never null?